Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -49,6 +47,8 @@ def family
           :icelake
         when 0x07d34b9f # ARMv8.3-A (Vortex, Tempest)
           :arm_vortex_tempest
+        when 0x573b5eec, 0x1b588bb3 # ARMv8.4-A (Firestorm, Icestorm)
```

Comment:
Is this two CPUs or one?

Reply:
Every CPU from this family features both Firestorm and Icestorm cores, see my explanation at https://github.com/Homebrew/brew/pull/9160#issuecomment-728890722.

The kernel uses two identifiers.
We don’t understand their difference (yet) but I think we shouldn’t care. One of them seems to be associated to the M1 MacBook Air, the other one to the M1 Macbook Pro and M1 Mac mini.

I suspect the kernel [differentiates between the passively/actively cooled variant](https://github.com/Homebrew/brew/issues/7857#issuecomment-728837841) for some internal purpose. But for Homebrew’s use case, they should be treated as one and the same.

Reply:
Gotcha 👍🏻